app/konnect/org-management/teams-roles-users.md

```diff
@@ -0,0 +1,195 @@
+
+To help secure and govern your environment, {{site.konnect_short_name}} provides
+the ability to manage authorization with teams and roles. You can use Konnect's
+predefined teams for a standard set of roles, or create custom teams with
+any roles you choose. Invite users and add them to these teams to manage user
+access.
+
+{:.note}
+> **Note:** If Okta integration is [enabled](/konnect/org-management/okta-idp),
+{{site.konnect_short_name}} users and teams become read-only. An organization
+admin can view all registered users in {{site.konnect_short_name}}, but cannot
+edit their team membership from the {{site.konnect_short_name}} side. To manage
+automatically-created users, adjust user permissions through Okta, or
+[adjust team mapping](/konnect/org-management/okta-idp/#map-teams-to-groups).
+
+## Prerequisites
+
+You must be part of the **Organization Admin** team to manage users, teams, and
+roles.
+
+## Manage teams and roles
+
+You can find a list of all teams in your organization through
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+**Organization** > **Teams** in Konnect.
+
+* **Team:** A group of users with access to the same roles. Teams can be
+used to functionally separate departments or create hierarchical
+structure in an organization. Access to Konnect resources such as runtime groups
+and services can be shared with teams. A team can be granted granular
+access based on permissions and roles.
+
+* **Role:** Predefined access to a particular resource type (for example,
+  Services).
+
+When you create a Konnect account, you are automatically added to the Organization
+Admin team, which is one of the [predefined teams](/konnect/org-management/teams-reference)
+in Konnect. Predefined teams have sets of roles that can't be modified or
+deleted. You can add users to these teams, or create your own custom teams
+with any of the [supported roles](/konnect/org-management/roles-reference).
+
+Users can be part of any number of teams, and the roles gained from the teams
+are additive. For example, if you add a user to both the Service Developer and
+Portal Viewer teams, the user can create and manage Services
+through the Service Hub _and_ register applications through the Dev Portal.
+
+All users can view all teams and members of each team, but they can't view the
+team permissions.
+
+### Entity and role sharing
+
+An organization admin can share any role or entity with any user in the
+organization.
+
+Any user can also share Services or runtime groups that they have access to, with
+the same role or lesser.
+
+For example, say you have a Service Admin role:
+* You can share that Service with any other user through the Service Hub.
+* Since you have admin permissions, you can choose to share the Service any
+level of access: creator, deployer, viewer, etc.
+* The user that you share this Service with can then share the Service
+themselves.
```

Comment:
This was de-scoped from helium.

Reply:
Yes, only this part has been de-scoped.
```
The user that you share this Service with can then share the Service
themselves.
```
If UserX is ServiceA Admin, and shares ServiceA Viewer to UserY. UserY cannot share viewer with anyone else.
Only ServiceA Admins will be allowed to share any roles for ServiceA to any other user or team.